ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/ACSReader.scala

```diff
@@ -185,7 +183,7 @@ private[events] object FilterTableACSReader {
     val queueState = new QueueState(signalQueue, initialTasks)
 
     signalSource
-      .mapAsyncUnordered(workerParallelism) { _ =>
+      .mapAsync(workerParallelism) { _ =>
```

Comment:
This is the whole real change in functionality.

Reply:
Can we have an in-code comment on the memory-limiting guarantees that the use of mapAsync is providing?